### PR TITLE
Move grafana dashboards

### PR DESCRIPTION
**Pull request by YuryHrytsuk**
## What do these changes do?
* move grafana dashboards to `./osparc-config`
* add `.shared_config.dir`

## Related issue/s
* resolves https://github.com/ITISFoundation/osparc-ops-environments/issues/229

## Related PR/s
* https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/505

## Checklist
- [X] I tested and it works

### CODE REVIEW

**Comment by mrnicegyu11** [comment]
I like a lot that you managed to move the assets, but we now have 3 "conenction" links (like `.config.location`). This scares me a bit, it should be sufficient to have one in my mind? Am I missing something?

**Comment by YuryHrytsuk** [comment]
> I like a lot that you managed to move the assets, but we now have 3 "conenction" links (like `.config.location`). This scares me a bit, it should be sufficient to have one in my mind? Am I missing something?

Hm, I think 2 because we have 2 configuration directories:
1. deployment-specific
2. shared / common

**Review by mrnicegyu11 — APPROVED** [review]
discussed in person, looks good as `.config.location` will be removed/refactored in a later step

**Comment by YuryHrytsuk** [comment]
@pcrespov, @mrnicegyu11 pointed out that you might be interested in the change we are doing to `.config.location`. As you can partly see in this PR, we introduce 2 new files: `.config.dir` and `.shrared_config.dir` which will substitute `.config.location`. 

All the paths to artifacts will be now hardcoded in a way:
```bash
config_dir=$(cat .config.dir)
repo_config="$config_dir"/repo.config
ansible_inventory="$config_dir"/ansible/inventory.ini
...
```

Let me/us know what you think of this change and be aware that `config.location` may cease to exist in near future.

**Comment by mrnicegyu11** [comment]
@pcrespov and in case you got scared because you read this before your first morning coffee, we want to make the integration of the config repos conventiant for the backend so this is all up for discussion and we can give an introduction once fundamental things, like the integration/connection of the repos, are changed 😗

File: services/monitoring/grafana/scripts/import.py [hunk 1]
```diff
@@ -448,10 +440,8 @@ def main(foldername: str = "", overwrite: bool = True):
     print("**************** Add alerts *******************")
     # Finally we import the dashboards
     if foldername == "":
-        directoriesAlerts = glob.glob(
-            "./../provisioning/" + env.str("MACHINE_FQDN") + "/alerts/*"
-        )
-        directoriesAlerts += glob.glob("./../provisioning/allDeployments" + "/alerts/*")
+        directoriesAlerts = glob.glob("./../assets/alerts/*")
```
Comment (pcrespov) [inline services/monitoring/grafana/scripts/import.py:443]: TIP: `pathlib` incorporates the functionality you are using in `glob`. e.g.
```python
assets_dir = Path("./../assets/")
assert assets_dir.is_dir()

config_file_path = assets_dir /datasources2dashboards.yaml")
directories_alerts = list( (assets_dir / "alerts").glob("*") ] 
```

**Review by pcrespov — APPROVED** [review]
Thx.

File: services/monitoring/grafana/scripts/import.py (hunk 1 above)
Comment (YuryHrytsuk) [reply services/monitoring/grafana/scripts/import.py:443]: Agree. This script (and others) need to be refactored but this is out of the scope of this PR and belongs to https://github.com/ITISFoundation/osparc-ops-environments/issues/189

**Pull request merged**